lib/normalizer.c

```diff
@@ -3488,6 +3488,19 @@ grn_nfkc_normalize_unify_iteration_mark(grn_ctx *ctx,
     return unified_buffer;
   }
 #  undef N_KATAKANA_BYTES
+#  define N_KANJI_BYTES 3
+  else if (current_length == 3 && current[0] == 0xe3 && current[1] == 0x80 &&
+           current[2] == 0xbb && previous_length == N_KANJI_BYTES &&
+           GRN_CHAR_TYPE(grn_nfkc_char_type(previous)) == GRN_CHAR_KANJI) {
+    /* U+303B VERTICAL IDEOGRAPHIC ITERATION MARK */
```

Comment:
Could you add a comment that describes unsupported cases?

Reply:
fix: [4c504f1](https://github.com/groonga/groonga/pull/2514/commits/4c504f14a8194a2d11b80d3dac80a1ab891bbd6f) Sure.